components/application/index.jsx

```diff
@@ -33,15 +32,17 @@ const Application = ({
         <h2 className="sr-only">Navigate your data</h2>
         {dataProvider && (
           <ActivitySelect>
-            {activities.map(({ id, path }) => (
-              <ActivitySelect.Option
-                dataProvider={dataProvider.id}
-                key={id}
-                value={id}
-                selected={id === activity.split('/')[0]}
-                path={path}
-              />
-            ))}
+            {activities.routes
+              .filter((activity) => activity.parent == null)
+              .map(({ path, test }) => (
+                <ActivitySelect.Option
+                  dataProvider={dataProvider.id}
+                  key={path}
+                  value={path}
+                  selected={test(pathname)}
+                  path={path}
```

Comment:
Maybe `value` or `path` is not needed anymore.

Reply:
one can be removed I guess.